CODEOWNERS

```
@@ -1 +1 @@
-*       @msporny @kdenhartog
+*       @msporny @dlongley @David-Chadwick @OR13 @selfissued @decentralgabe @awoie
```

```suggestion
*       @msporny @OR13 @selfissued @decentralgabe @awoie
```

Reducing to current list of Editor's listed on the spec.